main/glossary/README.md

```diff
@@ -87,51 +84,47 @@ or `BigInt(123)`.
+communication method&mdash;a DM or email or other private message, a phone call/voicemail,
+an email blast to a mailing list, publishing it on a website, etc.
 
 ## Brand
-Identifies the kind of [issuer](#issuer), such as "quatloos", "moola", etc. Brands are one of the two elements that
+Identifies the kind of [issuer](#issuer), such as "[Quatloos](#quatloos)",
+"[Moola](#moola)", etc. Brands are one of the two elements that
 make up an [amount](#amounts).
 For more information, see the [ERTP Guide's Brand section](/ertp/guide/amounts.md)
 and the [ERTP API's Brand section](/ertp/api/brand.md).
 
 ## Bundle
 
 Before a contract can be installed on Zoe, its source code must be bundled. This is done by:
-```
+```js
 import bundleSource from '@endo/bundle-source';
 const atomicSwapBundle = await bundleSource(
     require.resolve('@agoric/zoe/src/contracts/atomicSwap'),
 );
 ```
 The installation operation returns
 an `installation`, which is an object with one method; `getBundle()`. You can access an installed contract's source
-code via `const { source } = await E(installation).getBundle();`.
-In many cases, the bundled source is a single reviewable string.
-In others, the bundle contains to base 64 encoded zip file that you can
-extract for review.
-```
-jq -r .endoZipBase64 bundle.json | base64 -d > bundle.zip
+code via `const { endoZipBase64 } = await E(installation).getBundle();`.
+In most cases, the bundle contains a base64-encoded zip file that you can
+extract for review:
+```sh
+echo "$endoZipBase64" | base64 -d > bundle.zip
 unzip bundle.zip
 ```
 
 
 ## Burn
 
-Destroy all digital assets in a payment. See [`issuer.burn(payment, optAmount)`](/ertp/api/issuer.md#issuer-burn-payment-optamount).
+Destroy all digital assets in a payment, for example as part of consuming it in an exchange. See [`issuer.burn(payment, optAmount)`](/ertp/api/issuer.md#issuer-burn-payment-optamount).
 
 ## Comparable
 
-A *passable* is something that can be marshalled. A *comparable* is a
-passable whose leaves contain no promises. Two comparables can be
-synchronously compared for structural equivalence.
-
-A comparable is a JavaScript object containing no promises, and can
-thus be locally compared for equality with another object. If either object
-contains Promises, equality is indeterminable. If both are fulfilled down
-to Presences and local state, then either they're the same all the way
-down, or they represent different objects.
+A *comparable* is a [passable](#passable) containing no promises or errors, and can
+thus be synchronously compared for structural equivalence with another piece of data.
+If either side of the comparison contains promises and/or errors, equality is indeterminable.
+If both are fulfilled down to [presences](#presence) and local state, then either they're the
+same all the way down, or they represent different objects.
 
 ## Contract and Contract Instance
```

Comment:
```suggestion
## Contract Installation and Contract Instance
```
and use that terminology consistently below. I prefer to avoid synecdoche when speaking precisely.